development/gcp/cloud-run/search-github-issue/search-github-issues.go

```diff
@@ -56,6 +55,7 @@ func main() {
 	port = os.Getenv("LISTEN_PORT")
 	githubOrg = os.Getenv("GITHUB_ORG")
 	githubRepo = os.Getenv("GITHUB_REPO")
+	toolsGithubTokenPath = os.Getenv("TOOLS_SAP_TOKEN_PATH")
```

Comment:
please keep the name consistency:
ENV_NAME => envName
```suggestion
	port = os.Getenv("PORT")
	githubOrg = os.Getenv("GITHUB_ORG")
	githubRepo = os.Getenv("GITHUB_REPO")
	toolsGithubTokenPath = os.Getenv("TOOLS_GITHUB_TOKEN_PATH")
```

Or find different name for the env name. Let's try to keep the names short as possible. Maybe `GHE_TOKEN_PATH`, which translates to "GitHub Enterprise Token Path".

Reply:
You can't use PORT env var as it's collides with cloud run default env vars. You will get error when starting cloud run instance with such env definition.
I don't see anything wrong with longer variable names. Using shortcuts like GHE makes them harder to understand after some time. Longer names provide more information about what this variable is.